python/llm/example/GPU/PyTorch-Models/Model/llama2/README.md

```diff
@@ -67,3 +67,88 @@ What is AI?
 
 AI, or artificial intelligence, refers to the ability of machines to perform tasks that would typically require human intelligence, such as learning, problem-solving,
 ```
+
+## Example 2 - Low Memory Version: Predict Tokens using `generate()` API
+
+In the example [low_memory_generate.py](./low_memory_generate.py), we show a low memory use case for a Llama2 model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations on Intel GPUs.
+
+This method is recommended when you are running a model using [generate.py](./generate.py) but are out of memory.
+
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+pip install optimum # required by low memory version
+```
+
+### 2. Configures OneAPI environment variables
+```bash
+source /opt/intel/oneapi/setvars.sh
+```
+
+### 3. Run
```

Comment:
This entire usage is almost the same as original just with some extra params and an extra dependency. I think you can make this a complementary section with extra dependency and arguments specifications.

Reply:
Done.